rest/data/project_settings_test.go

```diff
@@ -637,16 +638,104 @@ func TestSaveProjectSettingsForSection(t *testing.T) {
 			}
 			apiSub := restModel.APISubscription{}
 			assert.NoError(t, apiSub.BuildFromService(newSubscription))
+
+			webhookSubscriber := restModel.APIWebhookSubscriber{
+				URL:    utility.ToStringPtr("http://example.com"),
+				Secret: utility.ToStringPtr("super_secret_2"),
+				Headers: []restModel.APIWebhookHeader{
+					{
+						Key:   utility.ToStringPtr("Key"),
+						Value: utility.ToStringPtr("A new value"),
+					},
+					{
+						Key:   utility.ToStringPtr("Authorization"),
+						Value: utility.ToStringPtr(evergreen.RedactedWebhookSecretsValue),
```

Comment:
Can we mention that this is testing a case where we're not actually updating the webhook, because this is a placeholder that the UI returns for the actual secret?

Reply:
I split up the test cases and aded comments